thyrel-web/src/components/home/ButtonModalJoin.tsx

```diff
@@ -27,10 +28,16 @@ export default function ButtonModalJoin({
   return (
     <>
       <BigButton
+        startIcon={
+          !urlIdentifier ? undefined : (
+            <PlayArrowIcon style={{ fontSize: 32 }} />
+          )
+        }
         fullWidth
         loading={loading}
         className={className}
         size="medium"
+        color={!urlIdentifier ? 'default' : 'primary'}
```

Comment:
ici aussi, on test tjr l'égalité quand on le peux
```suggestion
        color={urlIdentifier ? 'primary' : 'default'}
```